src/aind_codeocean_api/codeocean.py

```diff
@@ -443,6 +457,9 @@ def run_capsule(
             ]
             where position one refers to the parameter #1 ('input_folder'),
             parameter #2 ('output_folder'), and parameter #3 ('bucket_name')
+        custom_metadata : Optional[dict]
```

Comment:
This doesn't appear to be in the parameter list

Reply:
thanks, put this in the wrong spot